web/source/build.sh

```diff
@@ -276,6 +278,21 @@ readonly BUILD_DEBUG_EMBED
 readonly BUILD_COREWEB
 readonly DO_MINIFY
 
+generate_environment_ts_file ( ) {
+if [ -f $ENVIRONMENT_FILE ];
```

Comment:
Can you fix whitespace in this function please (obviously the multiline `echo` contents will still be unindented, but the rest could be better 🙂)

Reply:
Wasn't sure which way our conventions would dictate here.  Not a problem.